extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/discovery/ZonePlayerDiscoveryParticipant.java

```diff
@@ -29,6 +30,7 @@
  *
  * @author Karel Goderis - Initial contribution
  */
+@Component(name = "org.eclipse.smarthome.binding.sonos.discovery.zoneplayer", immediate = true)
```

Comment:
Afaik, we do not need a specific service name for that one, so I'd suggest to go for the default and remove that name property here.